tests/test_SpEC.py

```diff
@@ -9,28 +9,90 @@
 import h5py
 import scri
 import scri.SpEC
+import shutil
 
 
-@pytest.mark.skipif(not os.path.exists('../SpEC/samples/rhOverM_Asymptotic_GeometricUnits.h5'),
-                    reason="requires SpEC/samples/*.h5")
+# NOTE: test_file_io() depends on output from test_NRAR_extrapolation(), so the latter
+# must come first in this file.
+
+filename = "rh_FiniteRadii_CodeUnits.h5"
+input_dir = "../SpEC/samples/"
+output_dir_base = "../SpEC/samples/test_output/"
+
+
+@pytest.mark.skipif(
+    not os.path.exists(input_dir + filename),
+    reason="requires {}".format(input_dir + filename),
+)
+def test_extrapolation():
+    """
+    Test extrapolation of waveforms with output in the scri format.
+    """
+    output_dir = output_dir_base + "test_extrapolation_output/"
+    # Extrapolation fails if the output filenames exist so we delete
+    # the output files from a previous test if they exist.
+    if os.path.exists(output_dir):
+        shutil.rmtree(output_dir)
+
+    scri.extrapolation.extrapolate(
+        InputDirectory=input_dir, OutputDirectory=output_dir, ChMass=1.0
+    )
```

Comment:
Then, you can use the fixture like this:

```suggestion
def test_extrapolation(tempdir):
    """Test extrapolation of waveforms with output in the scri format"""
    input_dir = str(tempdir)
    output_dir = str(tempdir / "test_extrapolation")
    scri.extrapolation.extrapolate(InputDirectory=input_dir, OutputDirectory=output_dir, ChMass=1.0)
```
Note that there's no need to check if the input file exists; it will because the fixture created it.  No need to remove the output dir if it exists; it won't because the fixture created a new temporary directory.

Basically the same setup can be used for the next function, with `"test_extrapolation"` -> `"test_NRAR_extrapolation"`.